healthcheck.go

```diff
@@ -7,59 +7,65 @@ import (
 	"io/ioutil"
 	"net/http"
 
-	"github.com/Financial-Times/go-fthealth"
+	fthealth "github.com/Financial-Times/go-fthealth/v1_1"
+	"github.com/Financial-Times/service-status-go/gtg"
 	"github.com/Financial-Times/message-queue-gonsumer/consumer"
 )
 
 type healthcheck struct {
-	client       http.Client
-	consumerConf consumer.QueueConfig
+	client        http.Client
+	consumerConf  consumer.QueueConfig
+	appSystemCode string
 }
 
-func (h *healthcheck) healthcheck() func(w http.ResponseWriter, r *http.Request) {
-	return fthealth.HandlerParallel("Dependent services healthcheck", "Checks if all the dependent services are reachable and healthy.", h.messageQueueProxyReachable())
-}
-
-func (h *healthcheck) gtg(writer http.ResponseWriter, req *http.Request) {
-	healthChecks := []func() error{h.checkAggregateMessageQueueProxiesReachable}
-
-	for _, hCheck := range healthChecks {
-		if err := hCheck(); err != nil {
-			writer.WriteHeader(http.StatusServiceUnavailable)
-			return
-		}
+func (h *healthcheck) createHC() fthealth.HC {
+	return fthealth.HealthCheck{
+		SystemCode: h.appSystemCode,
+		Description: "Maps video's native format to FT's Content structure.",
+		Name: "video-mapper",
+		Checks: []fthealth.Check{h.messageQueueCheck()},
 	}
 }
 
-func (h *healthcheck) messageQueueProxyReachable() fthealth.Check {
+func (h *healthcheck) messageQueueCheck() fthealth.Check {
 	return fthealth.Check{
 		BusinessImpact:   "Publishing or updating videos will not be possible, clients will not see the new content.",
 		Name:             "MessageQueueProxyReachable",
 		PanicGuide:       "https://dewey.ft.com/up-vm.html",
 		Severity:         1,
 		TechnicalSummary: "Message queue proxy is not reachable/healthy",
-		Checker:          h.checkAggregateMessageQueueProxiesReachable,
+		Checker: func() (string, error) {
```

Comment:
why was wrapping `h.checkAggregateMessageQueueProxiesReachable` in a `func() {...}` needed?